### DISCUSSION TITLE
Distributed training in Spark

### DISCUSSION THREAD

**Discussion by rwforest** [pyg-team/pytorch_geometric#4361] [Q&A]
I get confused about the differences between the distributed training and capabilities in PyG vs Spark, there exists Horovord and Petastorm. If I process my data in Spark, can I use PyG in Spark to do disturbuted training?

**Comment by rusty1s** [comment]
Yes, please see my answer here: https://github.com/pyg-team/pytorch_geometric/issues/2529#issuecomment-1080309487
